tests/perf/test_semantic_segmentation.py

```diff
@@ -102,46 +102,46 @@ class TestPerfSemanticSegmentationSemiSL(TestPerfSemanticSegmentation):
     """Benchmark semantic segmentation."""
 
     MODEL_TEST_CASES = [  # noqa: RUF012
-        Benchmark.Model(task="semantic_segmentation", name="litehrnet_18_semisl", category="balance"),
-        Benchmark.Model(task="semantic_segmentation", name="litehrnet_s_semisl", category="speed"),
-        Benchmark.Model(task="semantic_segmentation", name="litehrnet_x_semisl", category="accuracy"),
-        Benchmark.Model(task="semantic_segmentation", name="segnext_b_semisl", category="other"),
-        Benchmark.Model(task="semantic_segmentation", name="segnext_s_semisl", category="other"),
-        Benchmark.Model(task="semantic_segmentation", name="segnext_t_semisl", category="other"),
-        Benchmark.Model(task="semantic_segmentation", name="dino_v2_semisl", category="other"),
+        Benchmark.Model(task="semantic_segmentation/semisl", name="litehrnet_18_semisl", category="balance"),
+        Benchmark.Model(task="semantic_segmentation/semisl", name="litehrnet_s_semisl", category="speed"),
+        Benchmark.Model(task="semantic_segmentation/semisl", name="litehrnet_x_semisl", category="accuracy"),
+        Benchmark.Model(task="semantic_segmentation/semisl", name="segnext_b_semisl", category="other"),
+        Benchmark.Model(task="semantic_segmentation/semisl", name="segnext_s_semisl", category="other"),
+        Benchmark.Model(task="semantic_segmentation/semisl", name="segnext_t_semisl", category="other"),
+        Benchmark.Model(task="semantic_segmentation/semisl", name="dino_v2_semisl", category="other"),
     ]
 
     DATASET_TEST_CASES = [  # noqa: RUF012
         Benchmark.Dataset(
             name="kvasir",
-            path=Path("semantic_seg/semisl/kvasir_24"),
+            path=Path("semantic_seg/semi-sl/kvasir_24"),
             group="small",
-            num_repeat=5,
-            unlabeled_data_path=Path("semantic_seg/semisl/unlabeled_images/kvasir"),
+            num_repeat=1,
```

Comment:
Why is it set to 1?

Reply:
All semisl has run_repeat=1, if not, only semisl testing will take over 10 days.

Reply:
I think, at least small datasets, increasing num_repeat is needed to check stable performance even though it takes a time.

Reply:
Can we keep only 1 dataset then?  It is not our priority and it uses same models as in supervised.

Reply:
@kprokofi - Isn't it better to measure performance once on 3 different splits of a dataset and average the results, rather than measuring performance 3 times on a single split, for the sake of diversity? When I previously discussed classification semi-sl datasets with Harim, I believe Harim is on the same page. Can you discuss this topic again with Harim in the OTX production group and let me know which approach is more efficient?